### PR TITLE
Git: Naming

### PR DESCRIPTION
**Pull request by mezis**
This addition covers naming branches, tagging commit messages, and titling PRs.

### CODE REVIEW

**Comment by kapowaz** [comment]
A couple of instances of ‘prefered’ which should be ‘preferred’ but otherwise LGTM 👍

File: _guidelines/git.md [hunk 1]
```diff
@@ -11,6 +11,23 @@ collection: guidelines
 1. Automatic Table of Contents Here
 {:toc}
 
+## Naming branches, commits, and pull-requests
+
+We orchestrate most of our work using a ticketing system (Jira). Assuming a
+ticket ID'd `AB-123` and titled "Install the Mr Fusion", 
+
+- The branch should be named `AB-123/install-the-mr-fusion`. The case of the
+  ticket ID and the slah are important.
```
Comment (kapowaz) [inline _guidelines/git.md]: > slah

Comment (mezis) [inline _guidelines/git.md]: You know about my hasty (and slightly dyslexic) typing by now 😁

File: _guidelines/git.md [hunk 2]
```diff
@@ -11,6 +11,23 @@ collection: guidelines
 1. Automatic Table of Contents Here
 {:toc}
 
+## Naming branches, commits, and pull-requests
+
+We orchestrate most of our work using a ticketing system (Jira). Assuming a
```
Comment (martyphee) [inline _guidelines/git.md]: If it's not in here already you might include a note about using [git-whistles](https://github.com/mezis/git-whistles).  It does everything for you.

Comment (mezis) [inline _guidelines/git.md]: Works for me, and obviously for Marty. If you're having issues I'll personally fix the gem :)

Comment (gregbeech) [inline _guidelines/git.md]: I think it was the 1.x version that started having issues. I just gave up after a while and reinstalled the old one without any of the Jira stuff. Maybe I'll try the new one again.

**Comment by gregbeech** [comment]
LGTM 👍

File: _guidelines/git.md (hunk 2 above)
Comment (mezis) [inline _guidelines/git.md]: (done)

File: _guidelines/git.md [hunk 3]
```diff
@@ -11,6 +11,28 @@ collection: guidelines
...
+  for commit messages.
+
+_Rationale_: having a consistent convention enables integrations. For instance,
+the above ensures that commits and PRs are reported into Jira, making it
+possible for someone to traverse links to read a story's implementation instead
+of searching.
+
+Note that [git-whistles](https://github.com/mezis/git-whistles) will do most of
+this for you: naming the branch right, the PR when issuing it, and even pasting
+the ticket description into your PR for better reviewr comfort!
```
Comment (lededje) [inline _guidelines/git.md]: Reviewer

**Comment by lededje** [comment]
lgtm

File: _guidelines/git.md (hunk 3 above)
Comment (krissy) [inline _guidelines/git.md]: Small typo: reviewr -> reviewer (otherwise :+1:!)

**Comment by mischizzle** [comment]
:shipit:

**Pull request merged**